src/lib/input/input-container.ts

```diff
@@ -263,6 +288,9 @@ export class MdInputContainer implements AfterContentInit {
   /** Whether the placeholder can float or not. */
   get _canPlaceholderFloat() { return this._floatPlaceholder !== 'never'; }
 
+  /** State of the md-hint and md-error animations. */
+  _messageAnimationState: string = '';
```

Comment:
How about something like `subscriptAnimationState` to specifically reference that it's underneath the input?